Sources/Paywalls/PaywallCacheWarming.swift

```diff
@@ -23,6 +23,9 @@ protocol PaywallCacheWarmingType: Sendable {
 
     @available(iOS 15.0, macOS 12.0, watchOS 8.0, tvOS 15.0, *)
     func warmUpPaywallFontsCache(offerings: Offerings) async
+
+    @available(iOS 15.0, macOS 12.0, watchOS 8.0, tvOS 15.0, *)
+    func triggerFontDownloadIfNeeded(fontsConfig: UIConfig.FontsConfig) async
```

Comment:
I wonder if we can remove the `async` here? This should be a `fire and forget` operation, so there's no need other wait